app/src/main/java/org/systers/mentorship/applicationClass.kt

```diff
@@ -0,0 +1,9 @@
+package org.systers.mentorship
+
+import org.systers.mentorship.models.User
```

Comment:
We don't need all , this instead we can just fetch the viewmodel/ fragment from the backstack.

Reply:
We can use backstack to fetch the data but once the profile is updated backstack will not work here as well as in whole application many times the user's information is fetched if we save the data in this application class the data can be fetched  throughout the application in no time without calling API

Reply:
Yes but we can even get the viewmodel from the backstack which would not call the api and still have the user profile details.

Reply:
As you say , this would work too but if you want me to use backstack instead i would commit the changes :)

Reply:
@epicadk I have used backstack but that is not working as supposed as it is calling API even if its not needed , Should i save the profile offline using Room database or should i stick to my previous commit ? Thanks for your help ! :-)